pkgs/tools/misc/pywb/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib, fetchFromGitHub, fetchPypi, python3, stdenv }:
+
+python3.pkgs.buildPythonApplication rec {
+  name = "pywb";
+  version = "2.7.4";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "webrecorder";
+    repo = "pywb";
+    rev = "v-${version}";
+    sha256 = "sha256-M44TwBzXuF33HNLB2OjmBKNDG9kt/qhGEOu589YGvsQ=";
+    # TODO: We clone the wombat submodule, but we still need to implement
+    # building wombat.
+    fetchSubmodules = true;
+  };
+
+  patches = [ ./pywb.patch ];
```

Comment:
See [comment](https://github.com/NixOS/nixpkgs/pull/189950#discussion_r1336993837). Please comment why this is needed and why it is not submitted upstream